source/MaterialXTest/ShaderValid.cpp

```diff
@@ -367,192 +365,68 @@ static void runValidation(const std::string& shaderName, std::vector<mx::Element
     }
 }
 
-TEST_CASE("GLSL geometry", "[shadervalid]")
+bool endsWithCaseInsensitive(std::string str, std::string toMatch)
 {
-#ifdef LOG_TO_FILE
-    std::ofstream logfile("log_shadervalid_glsl_geometry.txt");
-    std::ostream& log(logfile);
-#else
-    std::ostream& log(std::cout);
-#endif
-
-    mx::DocumentPtr doc = mx::createDocument();
-
-    mx::FilePath searchPath = mx::FilePath::getCurrentPath() / mx::FilePath("documents/Libraries");
-    loadLibraries({ "stdlib", "sxpbrlib" }, searchPath, doc);
-
-    // Create a graph testing some geometric nodes
-    mx::NodeGraphPtr nodeGraph = doc->addNodeGraph("geometry_attributes");
-
-    std::vector<mx::ElementPtr> outputList;
-
-    // Normal stream test
```

Comment:
Maybe we should remove these as we create data documents otherwise it's hard to tell what needs to be added.